### ISSUE TITLE
add traceroute for ipv4

### ISSUE THREAD

**Issue by multiSnow** [iputils/iputils#166]
Since `ping` and `tracepath` support both ipv4 and ipv6, it would be convenince to make `traceroute` also support ipv4, so that we don't need to install another 'networking tools suit' and carefully solve conflictions.
Thanks.

**Comment by kerolasa** [comment]
The iputils traceroute6 is deprecated, and not built by default. Quite frankly I don't see much point in polishing iputils traceroute6 to do same was what authoritative traceroute project[1] does better.

The authoritative traceroute projec is maintained by Dmitry Butskoy with most recent release 2016-03-08. I guess one possibility is to merge iputils and traceroute projects (with commit histories intact), and keep / grant all maintainers & developers appropriate access. But these sorts of merges require strong agreement a merge should be done, and getting everyone onboard can take long time. Commonly there is also question what is a benefit of merging projects when they can happily stay separate. Usual answer goes something along lines bigger communities get more contributions, and therefore stay more relevant and healthy even if some contributors or maintainers resign. But that does not convince everyone, or isn't even true in every conceivable situations.

To summarize. If something in project merge front is expected then getting all the people whose opinions needs to be heard talking in same thread is the first step. That might not lead to anywhere, but without discussion a merge is hostile takeover and I'm not doing that.

[1] http://traceroute.sourceforge.net/

**Comment by pevik** [comment]
I also don't see much point of having duplicate (and in our case half done) traceroute implementation (but some people are using it https://packages.debian.org/sid/inetutils-traceroute vs. https://packages.debian.org/sid/traceroute). But not sure if Dmitry Butskoy agreed about moving his implementation to iputils (traceroute does not even have public version control system).

**Comment by multiSnow** [comment]
Thanks for response.
I would try the original traceroute.
